lib/logqueue-fifo.c

```diff
@@ -539,7 +544,6 @@ log_queue_fifo_new(gint qoverflow_size, const gchar *persist_name)
     {
       INIT_IV_LIST_HEAD(&self->qoverflow_input[i].items);
       worker_batch_callback_init(&self->qoverflow_input[i].cb);
-      self->qoverflow_input[i].cb.user_data = self;
       self->qoverflow_input[i].cb.func = log_queue_fifo_move_input;
```

Comment:
I wouldn't move this initialization either, user data is const throughout the execution of this log queue instance. That's I think better reflected by leaving it here.